.gitignore

```diff
@@ -7,6 +7,7 @@
 Files.App.OpenDialog*.dll
 Files.App.SaveDialog*.dll
 Files.App.Launcher.exe
+Files.App.Launcher.exe.sha256
```

Comment:
Was this supposed to be added to the gitignore?

Reply:
Yeah it's an intermediate artifact produced by VS.

Reply:
@hez2010

Reply:
Sorry for the delay. Resolved.

Reply:
Thank you